cmake/syclcc-launcher

```diff
@@ -45,7 +45,7 @@ if __name__ == '__main__':
     if arg.startswith(CXX_COMPILER_ARG):
       cxx_compiler_exe = arg[len(CXX_COMPILER_ARG):]
     elif arg.startswith(SYCLCC_ARG):
-      syclcc_exe = arg[len(SYCLCC_ARG):]
+      syclcc_exe = arg[len(SYCLCC_ARG):].split("*")
```

Comment:
Can you please elaborate what this does?

Reply:
On windows you can't just call `syclcc`, but instead you need `python syclcc`. To accommodate for this, I added `python*${HIPSYCL_SYCLCC}` in `hipsycl-config.cmake.in`.
The asterisc is required, as we need this to be two different array elements for the subprocess launch, as otherwise `python syclcc` as a filename, would be searched for, which obviously does not exist, so we split at the `*`.
We can't just split at the whitespace, as `${HIPSYCL_SYCLCC}` might be a full path, which on Windows might contain spaces not too uncommonly .. so I chose a character that is not allowed in paths.

Reply:
Ah, I see. I guess we can leave it like that, but maybe add a comment?

Reply:
sure :)